spec/models/manageiq/providers/kubernetes/container_manager/container_group_spec.rb

```diff
@@ -11,4 +11,33 @@
     )
     expect(group.container_images.count).to eq(1)
   end
+
+  # check https://bugzilla.redhat.com/show_bug.cgi?id=1406770
+  it "has container volumes as parent" do
+    group = FactoryGirl.create(
+      :container_group,
+      :name => "group",
+    )
+
+    ems = FactoryGirl.create(
+      :ems_kubernetes,
+      :id   => group.id,
+      :name => "ems"
+    )
+
+    FactoryGirl.create(
+      :container_volume,
+      :name   => "group_child",
```

Comment:
consider naming it "container_volume".  you're *hoping* it will be the pod's and only the pod's child but let's not mix expectations with facts.